automatic/brave/README-beta.md

```diff
@@ -10,4 +10,6 @@ Brave is a free and open-source web browser developed by Brave Software Inc. bas
 
 ## Notes
 
-* Beta is an early preview for new versions of Brave. This build showcases the newest advances and it’s ready for daily use. Brave Beta automatically sends crash reports, but you can turn this off if you’d like.
+- Beta is an early preview for new versions of Brave. This build showcases the newest advances and it’s ready for daily use. Brave Beta automatically sends crash reports, but you can turn this off if you’d like.
+
```

Comment:
Please remove this empty line, as it won't show up as expected once pushed to CCR.